cli/tests/end_to_end/extension.rs

```diff
@@ -5,7 +5,7 @@ use crate::common::{create_lockfile, create_project, TestCli};
 /// Test Phylum API functions.
 #[tokio::test]
 pub async fn get_user_info() {
-    let test_cli = TestCli::builder().with_config(true).build();
+    let test_cli = TestCli::builder().with_config(None).build();
```

Comment:
I would rather have something like `Default::default()` here, as `None` intuitively means "with no config", while the semantics of this call are "with a config, but don't specify which".

If we don't need to create a config, we can just skip the `.with_config()` invocation altogether.

Reply:
> If we don't need to create a config, we can just skip the .with_config() invocation altogether.

Yeah that's exactly why I chose `None` as parameter. These are tests and brevity is somewhat important. So I think having to specify the config every time would be annoying, especially because `Config::default` will not work.

Reply:
How about adding `.with_default_config()` or similar name (maybe shorter)? It's an extra one-liner method but it would be used in most places.

Reply:
It wouldn't be the default config though? I'm not sure tbh, I don't like having two different methods like this.

Reply:
I'd say this is not a dealbreaker anyway. We can keep the current solution.